### PR TITLE
Better beer countdown text

### PR DESCRIPTION
**Pull request by githuib**
Leave out hours / minutes when they are zero.

### CODE REVIEW

**Comment by Marfru** [comment]
Has this been tested? Please assign to tester @githuib

**Comment by githuib** [comment]
@Marfru ByndieBot can test itself.

File: plugins/beertime.py [hunk 1]
```diff
@@ -91,14 +77,18 @@ def get_beer_slogan():
...
+    elif todaybeeroclock <= now < todayendbeeroclock:
+        return "(jesse) {}".format(random.choice(beertimeslogans))
     else:
-        return "(jesse) {}just wait another {} hours {} minutes and {} seconds".format(
-            tooearlyslogans[randint(0, len(tooearlyslogans) - 1)], h, m, s)
+        s = formatcountdown.seconds
+        h, s = divmod(s, 3600)
+        m, s = divmod(s, 60)
+
+        hours = '{} hours '.format(h) if h > 0 else ''
```
Comment (joshp123) [inline plugins/beertime.py]: ternary operators aaaaaaaaaaaaaag

Comment (githuib) [inline plugins/beertime.py]: haters gonna hate

Comment (githuib) [inline plugins/beertime.py]: I did it for @lennartwesdijk

**Comment by joshp123** [comment]
thanks for changing @githuib you a bro

**Pull request merged**